### ISSUE TITLE
Installing Signal Desktop on Raspberry Pi

### ISSUE THREAD

**Issue by kyderby-code** [snapcrafters/signal-desktop#38]
I went on the snap.io forum initially, and it was suggested that I open an issue here at Github.  I did sudo snap install --edge signal-desktop on my Raspberry Pi running raspbian, then got this error:

/snap/signal-desktop/297/snap/command-chain/desktop-launch: line 543: /snap/signal-desktop/297/opt/Signal/signal-desktop: cannot execute binary file: Exec format error
/snap/signal-desktop/297/snap/command-chain/desktop-launch: line 543: /snap/signal-desktop/297/opt/Signal/signal-desktop: Success

Thanks to the community for any help.

**Comment by blackfireburn** [comment]
the same problem with pinebook pro. it lets me install using edge and then the same error as above.

**Comment by jpo234** [comment]
I downloaded the Snap information with 
`curl -H 'Snap-Device-Series: 16' http://api.snapcraft.io/v2/snaps/info/signal-desktop` and then downloaded the actual snap file:
```
    {
      "channel": {
        "architecture": "arm64",
        "name": "edge",
        "released-at": "2020-01-24T22:34:50.975065+00:00",
        "risk": "edge",
        "track": "latest"
      },
      "created-at": "2020-01-24T22:33:24.417371+00:00",
      "download": {
        "deltas": [],
        "sha3-384": "c0759508df5d75941d94cd6c6ef9c3c213fbf047fd3f0d8d1729bf203a763a69340714c3203cec9e43956b8d598b6348",
        "size": 131563520,
        "url": "https://api.snapcraft.io/api/v1/snaps/download/r4LxMVp7zWramXsJQAKdamxy6TAWlaDD_296.snap"
      },
      "revision": 296,
      "type": "app",
      "version": "1.30.0"
    },
```
And the binary in the package for the architecture arm64 is actually:
```
file /snap/signal-desktop/296/opt/Signal/signal-desktop 
/snap/signal-desktop/296/opt/Signal/signal-desktop: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=cc24f25f5633f1b4ac1a156a7d84fc2e363bf4ee, stripped
```

**Comment by galgalesh** [comment]
duplicate of https://github.com/snapcrafters/signal-desktop/issues/27
